src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeClientTests.cs

```diff
@@ -257,7 +257,7 @@ public async Task Close_FailsToCallClose_EslintBridgeProcessStillStopped()
             return new TypeScript.EslintBridgeClient.EslintBridgeClient(AnalyzeEndpoint, eslintBridgeProcess, httpWrapper, analysisConfiguration);
         }
 
-        private static Mock<IEslintBridgeHttpWrapper> SetupHttpWrapper(string endpoint, string response = null, Action<object> assertReceivedRequest = null)
+        private static Mock<IEslintBridgeHttpWrapper> SetupHttpWrapper(string endpoint, string response = "OK!", Action<object> assertReceivedRequest = null)
```

Comment:
Because of the bug that Rita fixed, we weren't throwing when there was an error. Now we have fixed the issue, the tests started to fail because of a null response.